### ISSUE TITLE
Bug: anchor links are not keyboard or screen reader accessible

### ISSUE THREAD

**Issue by andrewhick** [alphagov/wcag-primer#131]
The link icon which appears next to each heading on hover is not keyboard accessible, or presented to screen reader users.

We discussed this as a group and decided that although the functionality is potentially a useful enhancement for some people, it would be hard to make accessible and the easiest course of action is to remove it if we can. For comparison, GOV.UK does not have this functionality.

If we cannot fix it then we should flag this with the Tech Docs repository.

Detected by @shabana-ali and Lori.

This fails WCAG 2.1.1 Keyboard and probably 4.1.2 Name, Role, Value.

**Comment by andrewhick** [comment]
Discussed with the Tech Docs team who recommend that this is fixed in that repo. I've raised this as an issue there but suggest keeping this one open for visibility for now: https://github.com/alphagov/tech-docs-gem/issues/402
